### PR TITLE
Add a confirm prompt when deleting schedules

### PR DESCRIPTION
**Pull request by borkweb**
See: Findings doc

### CODE REVIEW

File: src/resources/js/aggregator-fields.js [hunk 1]
```diff
@@ -86,13 +86,14 @@ tribe_aggregator.fields = {
...
-			.on( 'click'      , obj.selector.clear_filters_button    , obj.clear_filters )
-			.on( 'click'      , obj.selector.finalize_button         , obj.finalize_manual_import )
-			.on( 'click'      , '.tribe-preview'                     , obj.preview_import )
-			.on( 'click'      , '.tribe-cancel'                      , obj.events.cancel_edit )
-			.on( 'change'     , obj.selector.origin_field            , function() {
+			.on( 'keypress'   , obj.selector.fields                    , obj.events.trigger_field_change )
+			.on( 'click'      , obj.selector.save_credentials_button   , obj.events.trigger_save_credentials )
+			.on( 'click'      , obj.selector.clear_filters_button      , obj.clear_filters )
+			.on( 'click'      , obj.selector.finalize_button           , obj.finalize_manual_import )
+			.on( 'click'      , '.tribe-preview'                       , obj.preview_import )
```
Comment (bordoni) [inline src/resources/js/aggregator-fields.js]: Can we store these selectors?

Comment (borkweb) [inline src/resources/js/aggregator-fields.js]: NO!

Comment (borkweb) [inline src/resources/js/aggregator-fields.js]: Sure. I'll make that change.

**Comment by borkweb** [comment]
This is ready for another look, Gustavo.

**Comment by barryhughes** [comment]
:meat_on_bone:

**Pull request merged**